Salesforce_Advanced/Work_Dot_Com_Command_Center/README.md

```diff
@@ -0,0 +1,94 @@
+
+Salesforce has responded to the Covid-19 crisis by adding a significant amount of functionality to the Work.com offering.  The Workplace Command Center they quickly rolled out presents a very interesting set of capabilities and starting points for developing effective tools for employers who are managing a crisis of this nature. It helps employers answer these questions: 
+
+- Are all my employment locations available to be opened? 
+- Are my employees at risk?  Have they been exposed?  Are they presenting symptoms?  Should they come to work today? 
+- Have my facilities received the necessary cleaning to mitigate risk of infection there? 
+
+As should be expected,  Skuid works really well with the new sObjects and relationships that have been rolled out in this solution.  We'd venture that Skuid is the perfect tool for adapting the Work.com Command Center offering to the real-world needs of your company. 
+
+This page will host a number of Skuid extensions to the Work.com solution. 
+
+<img src="WorkDotCom_CommandCenter.png" width="300"></img>
+
+## Getting Started. 
+### 1. Install Work.com command center  
+The easiest way to get started exploring the Work.com solution is to install this in a scratch org using SFDX.  This means you will need to use your terminal and have git and SFDX installed.  [Read these instructions on installing SFDX](https://trailhead.salesforce.com/en/content/learn/modules/sfdx_app_dev/sfdx_app_dev_setup_dx#Tdxn4tBK-heading7)
+
+Then follow these instructions: 
+- Clone a local copy of the [WorkDotCom-Partners repo](https://github.com/forcedotcom/WorkDotCom-Partners) and then `cd` into its directory
+
+ `git clone https://github.com/forcedotcom/WorkDotCom-Partners.git`
+
+- Run the orgInit.sh file which will use SFDX to install all the necessary packages and metadata.   
+
+`./orgInit.sh`
+
+Now you can see the basic Work.com solution. 
+
+More information: 
+- [Help Pages.](https://help.salesforce.com/articleView?id=workdotcom.htm&type=5) Overview of the solution and additional installation options. 
+- [Developer Docs.](https://developer.salesforce.com/docs/atlas.en-us.workdotcom_dev_guide.meta/workdotcom_dev_guide/wdc_cc_overview.htm) Additional configuration of this solution. 
+- [Data Model.](https://developer.salesforce.com/docs/atlas.en-us.workdotcom_dev_guide.meta/workdotcom_dev_guide/wdc_cc_data_model.htm) Understanding the Data Model will be particularly important for any Skuid development. 
+
+### 2.  Configure Consent Management. 
+Parts of the Work.com solution is not automatically configured in step 1.  The critical one for the Skuid solution is consent management.  [Follow these steps](https://help.salesforce.com/articleView?id=wcc_setup_configure_consent_management.htm&type=5) 
+
+After creating the `Authorization Form Text` record - copy it's  `ID` field for use in the Skuid page found below.   It is most easily viewed in the URL of the detail page.  It will look like `0cN1700000003KiEAI`
```

Comment:
```suggestion
After creating the `Authorization Form Text` record - copy its  `ID` field for use in the Skuid page found below.   It is most easily viewed in the URL of the detail page.  It will look like `0cN1700000003KiEAI`
```